src/test/java/net/serenitybdd/starter/fetests/testCases/BaseTest.java

```diff
@@ -0,0 +1,44 @@
+package net.serenitybdd.starter.fetests.testCases;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import net.serenitybdd.starter.model.User;
+import net.thucydides.core.annotations.Managed;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BaseTest {
+
+    @Managed
+    WebDriver driver;
+
+    public void startApp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.get("https://www.saucedemo.com");
+    }
+
+    public void stopApp() {
+        driver.quit();
+    }
+
+    public void logIn(User user) {
+        if(!driver.getCurrentUrl().equals("https://www.saucedemo.com")) driver.get("https://www.saucedemo.com");
```

Comment:
method relates to PageObject and Steps not to Test. Please move to corresponding layer